requirements.txt

```diff
@@ -1,3 +1,5 @@
-Django>=1.3
+Django==1.8
 geopy==0.94.2
 python-geohash==0.8.5
+flake8==2.5.1
+coverage==4.0.3
```

Comment:
Yes - I've separated them because tox has install different versions on django. I guess we also need them in the main `requirements.txt` for development.

**Update**
I've now removed them from the main file and included them into `requirements.txt` recursively,.